WISE-Community/WISE-Client
# chore(Authoring): Convert node authoring component to Angular

## Changes

- Converted node authoring component to Angular
- Converted component advanced dialog to Angular

## Test

Make sure node authoring works including
- Adding a new component
- Importing a component
- Moving a component
- Copying a component
- Deleting a component
- Editing the step rubric
- Step advanced
  - General Advanced
  - Edit Transitions
  - Edit Constraints
  - Create Branch
  - Show JSON
- Undo
- Preview step
- Changing step icon
- Changing step title
- Expand All
- Collapse All
- Component advanced dialog
- Preview component
- Expand/collapse single component
- Editing a component saves changes

Closes #478
